packages/aggregator/src/tests/analytics/aggregateAnalytics/aggregations/sumAcrossPeriods.test.js

```diff
@@ -123,5 +130,41 @@ describe('sumAcrossPeriods', () => {
         );
       });
     });
+
+    describe('excludeFuture', () => {
+      it('defaults to `false`', () => {
+        expect(sumAcrossPeriods(ANALYTICS, { periodOptions: { periodOptions: {} } })).to.deep.equal(
```

Comment:
```suggestion
        expect(sumAcrossPeriods(ANALYTICS, { periodOptions: {} })).to.deep.equal(
```
And similarly below.

Reply:
Great catch!! 👀

Reply:
Fixed in https://github.com/beyondessential/tupaia/pull/1012/commits/7d3fbebb19a1fc5677f439cf6a24d0a89055eb13 - I couldn't find another instance of that issue, but please let me know if I'm wrong